inst/extdata/PA2024CH_en_exec_summary/scorecard.Rmd

```diff
@@ -227,206 +227,135 @@ data_prep_net_zero_commitments <- prep_net_zero_commitments(
 ```
 
 ```{=latex}
-\subsection{Overall Climate Strategy including Credible Climate Stewardship}
+\subsection{Management to Net-Zero}
 
-Financial institutions can contribute to the transition to net-zero, by engaging with invested companies on third-party verified, science-based net-zero aligned transition plans until 2050.
+Financial institutions can contribute to the transition to net-zero, by aligning their investment strategy with a consistent 1.5\textdegree C
+decarbonisation pathway.
 
 ```
 
-```{r climate_strategy_initiatives}
-if (file.exists(file.path(score_card_dir, "climate_scorecard_survey_initiatives.csv"))) {
-  user_climate_strategy_scorecard_initiatives <- readr::read_csv(
-    file.path(score_card_dir, "climate_scorecard_survey_initiatives.csv"),
+```{r management_to_net_zero}
+data_net_zero_available <- file.exists(file.path(score_card_dir, "management_to_net_zero.csv"))
+if (data_net_zero_available) {
```

Comment:
Do you not need an `else` statement here to handle the case that there is no data available? 
Or is that not a problem?